detections/endpoint/windows_obfuscated_files_or_information_via_rar_sfx.yml

```
@@ -0,0 +1,68 @@
+name: Windows Obfuscated Files or Information via RAR SFX
+id: 4ab6862b-ce88-4223-96c0-f6da2cffb898
+version: 1
+date: '2024-12-12'
+author: Teoderick Contreras, Splunk
+data_sources:
+- Sysmon Event ID 11
+type: TTP
+status: production
+description: The following analytic detects the creation of RAR Self-Extracting (SFX) files by monitoring the generation of file related to rar sfx installation. This method leverages a heuristic to identify RAR SFX archives based on specific markers that indicate a combination of executable code and compressed RAR data. By tracking such activity, the analytic helps pinpoint potentially unauthorized or suspicious file creation events, which are often associated with malware packaging or data exfiltration. Legitimate usage may include custom installers or compressed file delivery.
+kind: endpoint
+search: '`sysmon` EventCode=11 Image="*.exe" TargetFilename IN ("*__tmp_rar_sfx_access_check*") 
```

You don't need the `Image="*.exe"` as the filename in itself is indicative.
 
```suggestion
search: '`sysmon` EventCode=11 TargetFilename IN ("*__tmp_rar_sfx_access_check*") 
```

---

thanks man! ;)